Tests/JSONUtilitiesTests/InlineDecodingTests.swift

```diff
@@ -78,6 +78,24 @@ class InlineDecodingTests: XCTestCase {
     }
   }
 
+  func test_decodingMandatoryEnumDictionary_withKey() {
+    let dictionary: JSONDictionary = ["enums": ["value1": "one", "value2": "!@1", "value3": "two"]]
+
+    let decodedEnums: [String: MockParent.MockEnum] = try! dictionary.json(atKeyPath: "enums")
```

Comment:
Makes sense, at some point we should clean this things, something for the future...